x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/ConnectorIndexService.java

```diff
@@ -314,6 +324,33 @@ public void onFailure(Exception e) {
         }
     }
 
+    /**
+     * Constructs a query for filtering connectors based on index and/or connector names.
+     * Returns a {@link MatchAllQueryBuilder} if both parameters are empty or null,
+     * otherwise constructs a boolean query to filter by the provided lists.
+     *
+     * @param indexNames List of index names to filter by, or null/empty for no index name filtering.
+     * @param names List of connector names to filter by, or null/empty for no name filtering.
+     * @return A {@link QueryBuilder} tailored to the specified filters.
+     */
+    private QueryBuilder buildListQuery(List<String> indexNames, List<String> names) {
```

Comment:
```suggestion
    private QueryBuilder buildListQuery(List<String> indexNames, List<String> connectorNames) {
```

We could make this explicit as we're having two filter params, which are have something to do with `...names`, up to you :) Considering that the code is explicit about it in the boolean `filterByConnectorNames` it could make sense. (This should probably be reflected then also in the actual params for the external facing API)

Reply:
Sure, I agree updating :)